samples/acceptance_tests.ged

```diff
@@ -430,7 +430,7 @@
 1 WIFE @I7@
 1 CHIL @I16@
 1 MARR
-2 DATE 4 OCT 2020
+2 DATE 4 DEC 2020

```

Comment:
This is in the future....pretty sure there is some kind of test that says marriage should happen before current date. Just make sure if you change this one that it doesn't break one of the validation tests above otherwise you will have to just add new people entirely.

Reply:
I looked there was no user story for marriage in the future

Reply:
User story 1: Dates (birth, marriage, divorce, death) should not be after the current date
This is why Family 1 shows up in validation messages for errors.